app/views/goals/show.html.erb

```diff
@@ -0,0 +1,89 @@
+<div>
+  <h2 class="titles">Goal: <%= @goal.title %></h2>
+  <div class="goal-container gbox">
+    <div id="goal-info" class="goal label">
+      <h4 class="attribute-label">Set by <%= @goal.setter.username %></h4>
+      <% if @goal.description != "" %>
+        <p><%= @goal.description %></p>
+      <% end %>
+      <p class="styled">
+        <meter min="0" max="100" low="25" high="75" optimum="100" value="20"></meter>
+      </p>
+      <p class="attribute-match"><%= @goal.setter.username %> has pledged $<%= @goal.setter_pledge %> towards completing this goal.</p>
+      <% if @goal.is_fully_funded? %>
+        <p>Supporters have made <%= @goal.pledges.count %> pledges to fully match <%= @goal.setter.username %>'s pledge.</p>
+      <% else %>
+        <p>$<%=@goal.left_to_match %> of <%= @goal.setter.username %>'s pledge remains to be matched by supporters.</p>
+      <% end %>
+      <p id="charity">Benefitting <%= link_to "#{@goal.charity.name}", charity_path %></p>
+      <% if @goal.tender %>
+        <p class="attribute-tender">Goaltender: <%= @goal.tender.username %></p>
+      <% end %>
+    </div>
+
+    <% if current_user == @goal.setter %>
+    <div id="options">
+      <ul>
+        <li class="option-links"><%= link_to "Edit Goal", edit_goal_path %></li>
+        <!-- li class="option-links"><%= link_to "Delete Goal", goal_path, :method => :delete %></li -->
+      </ul>
+    </div>
+    <% end %>
+
+    <% if current_user && @goal.setter != current_user && !@goal.is_fully_funded? %>
+    <div class="support-button">
+      <%= button_to "Support", new_goal_pledge_path(@goal),:method => :get%>
+    </div>
+    <% end %>
+  </div>
+
+  <h3 class="titles">Milestones</h3>
+  <div class="milestone-container cbox">
+    <ul>
+      <% @milestones.each do |milestone| %>
+        <li>
+          <p>
+            <%= milestone.description %>
+            <% if milestone.confirmed %>
+              &bull; <b>Completed <%= std_format_day milestone.updated_at %></b>
+            <% else %>
+              &bull; Deadline <%= std_format_day milestone.deadline %>
+            <% end %>
+          </p>
+          <% if @goal.setter == current_user && milestone.completed == false %>
+            <p>Goalsetter: <%= link_to "Notify goaltender of milestone completion", edit_goal_milestone_path(goal_id: @goal.id, milestone_id: milestone.id) %></p>
+          <% end %>
+          <% if @goal.tender == current_user && milestone.completed == true && milestone.confirmed == false %>
+            <p>Goaltender: <%= link_to "Confirm completion of milestone", edit_goal_milestone_path(goal_id: @goal.id, milestone_id: milestone.id) %></p>
+          <% end %>
+
+        </li>
+      <% end %>
+    </ul>
+  </div>
+
```

Comment:
Pull some of these conditionals into the model classes. For example
<% if current_user && @goal.setter != current_user && !@goal.is_fully_funded? %>

could be 
if @goal.supportable_by?(curren_user)

On goal
def supportable_by?(user)
  !(user == setter || is_fully_funded)
end